build.assets/Makefile

```diff
@@ -150,8 +137,7 @@ buildbox:
 		--build-arg GOGO_PROTO_TAG=$(GOGO_PROTO_TAG) \
 		--build-arg NODE_GRPC_TOOLS_VERSION=$(NODE_GRPC_TOOLS_VERSION) \
 		--build-arg NODE_PROTOC_TS_VERSION=$(NODE_PROTOC_TS_VERSION) \
-		--build-arg PROTOC_VERSION=$(PROTOC_VERSION) \
-		--cache-to type=inline \
+		--build-arg PROTOC_VER=$(PROTOC_VER) \
```

Comment:
Is this intentional?

Reply:
Yeah, it looks like `--build-arg PROTOC_VERSION=$(PROTOC_VERSION) \` was another bug that slipped through in the original backport.